controllers/digital-funding/views/strand-1.njk

```diff
@@ -0,0 +1,173 @@
+{% from "components/hero.njk" import hero %}
+{% from "components/breadcrumb-trail/macro.njk" import breadcrumbTrail %}
+{% from "components/content-box/macro.njk" import contentBox %}
+{% from "components/segment-links/macro.njk" import segmentLinks %}
+{% from "./view-helpers.njk" import steps, getStarted %}
+
+{% extends "layouts/main.njk" %}
+{% set pageAccent = 'blue' %}
+
+{% block content %}
+    <main role="main">
+        {{ hero(
+            titleText = title,
+            image = heroImage,
+            accent = pageAccent
+        ) }}
+
+        <div class="nudge-up">
+            <section class="content-box u-inner-wide-only accent--{{ pageAccent }} a--border-top">
+                {{ breadcrumbTrail(breadcrumbs) }}
+
+                <div class="s-prose u-constrained-content-wide">
+                    <p>Big Lottery Fund offers grants of up to £500,000, plus a package of tailored support, to help established established charities that want to use digital to help take a major leap forward.</p>
+                </div>
+
+                {{ segmentLinks([{
+                    "title": "How the process works"
+                }, {
+                    "title": "Success factors"
+                }, {
+                    "title": "Eligibility criteria"
+                }] )}}
+            </section>
+
+            <section id="segment-1" class="content-box u-inner-wide-only accent--{{ pageAccent }} a--border-top">
+                <div class="content-sidebar">
+                    <div class="content-sidebar__primary">
+                        <div class="s-prose">
+                            <h2>How the process works</h2>
+                            <p>Intro sentence or two to the conversational approach this fund will take and how it’s designed to help the customer experience.</p>
```

Comment:
Maybe stick a `@TODO` here so we can catch this before it inevitably goes live by accident...